src/audio/Audio.d.ts

```diff
@@ -1,6 +1,7 @@
 import { Object3D } from './../core/Object3D';
 import { AudioListener } from './AudioListener';
 import { AudioContext } from './AudioContext';
+import { AudioBuffer } from "./PositionalAudio";
```

Comment:
How about moving the `AudioBuffer` to `Audio.d.ts`? It seems it is now used in `PositionalAudio`.

Reply:
Sure. I wasn't sure why it was there but didn't want to make any assumptions. I'll fix it up now.

Reply:
@Mugen87 Okay. Should be good to go now. I moved the definition for AudioBuffer into the Audio.d.ts file and removed the import to PositionalAudio.

Reply:
Whoops... my Prettier kicked in - I'll switch back to match existing style.

Reply:
Okay... used this: http://zz85.github.io/mrdoobapproves/
Hope that makes this style compliant now.